src/app/ogr/qgsvectorlayersaveasdialog.h

```diff
@@ -65,6 +65,36 @@ class QgsVectorLayerSaveAsDialog : public QDialog, private Ui::QgsVectorLayerSav
 
     bool onlySelected() const;
 
+    /** Returns the selected flat geometry type for the export.
+     * @see automaticGeometryType()
+     * @see forceMulti()
+     * @see includeZ()
+     */
+    QgsWKBTypes::Type geometryType() const;
+
+    /** Returns true if geometry type is set to automatic.
+     * @see geometryType()
+     */
+    bool automaticGeometryType() const;
+
+    /** Returns true if force multi geometry type is checked.
+     * @see includeZ()
+     */
+    bool forceMulti() const;
+
+    /** Sets whether the force multi geometry checkbox should be checked.
+     */
+    void setForceMulti( bool checked );
+
+    /** Returns true if include z dimension is checked.
+     * @see forceMulti()
+     */
+    bool includeZ() const;
+
+    /** Sets whether the include z dimension checkbox should be checked.
+     */
+    void setIncludeZ( bool checked );
```

Comment:
There was briefly... Until I realized that OGR has no support for m values, so it was useless ;)